src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -247,20 +247,22 @@
                      {chain-id           :chain-id
                       network-value-type :type
                       total-amount       :total-amount}]
-                  (let [status (cond (and (= network-value-type :not-available)
-                                          loading-routes?
-                                          token-not-supported-in-receiver-networks?)
-                                     :loading
-                                     (= network-value-type :not-available)
-                                     :disabled
-                                     :else network-value-type)]
+                  (let [status           (cond (and (= network-value-type :not-available)
+                                                    loading-routes?
+                                                    token-not-supported-in-receiver-networks?)
+                                               :loading
+                                               (= network-value-type :not-available)
+                                               :disabled
+                                               :else network-value-type)
+                        amount-formatted (-> (rf/sub [:wallet/send-amount-formatted total-amount])
+                                             (str " " token-symbol))]
```

Comment:
using consistent formatting of the values based on the calculated `token-decimals`. Otherwise the `total-amount` can be shown as bigger than the `out-amount` due to rounding, which is wrong.

Reply:
@clauxx unless it's worse in terms of code strucuture, I'd prefer that the subscription already returned the whole computed value, so that we avoid calculations on the view.